x-pack/plugins/security_solution/public/flyout/right/components/description.test.tsx

```diff
@@ -33,21 +38,34 @@ const ruleDescription = {
   originalValue: ['description'],
   isObjectArray: false,
 };
+const ruleName = {
+  category: 'kibana',
+  field: 'kibana.alert.rule.name',
+  values: ['rule-name'],
+  originalValue: ['rule-name'],
+  isObjectArray: false,
+};
+
+jest.mock('../../../common/lib/kibana');
+jest.mock('../../../common/components/link_to');
 
 describe('<Description />', () => {
   it('should render the component collapsed', () => {
     const panelContextValue = {
-      dataFormattedForFieldBrowser: [ruleUuid, ruleDescription],
+      dataFormattedForFieldBrowser: [ruleUuid, ruleDescription, ruleName],
     } as unknown as RightPanelContext;
 
     const { getByTestId } = render(
       <RightPanelContext.Provider value={panelContextValue}>
-        <Description />
+        <ThemeProvider theme={mockTheme}>
+          <Description />
+        </ThemeProvider>
       </RightPanelContext.Provider>
     );
 
     expect(getByTestId(DESCRIPTION_TITLE_TEST_ID)).toBeInTheDocument();
     expect(getByTestId(DESCRIPTION_TITLE_TEST_ID)).toHaveTextContent(RULE_DESCRIPTION_TITLE);
+    expect(getByTestId(DESCRIPTION_TITLE_TEST_ID)).toHaveTextContent(VIEW_RULE_TEXT);
```

Comment:
similar to my comment on the Cypress file, you could also target directly the element that has the `data-test-subj='ruleName'` here to highlight the distinction